start.py

```diff
@@ -9,8 +9,13 @@
 # increase the number if your internet connecion is slow
 settings.browser_speed = 1.0
 
-
 kingbot = king_bot(email=email, password=password,
-                   gameworld=gameworld, proxy=proxy, start_args=sys.argv)
+                   gameworld=gameworld, proxy=proxy, start_args=sys.argv, debug=True)
 
 # place your actions below
+#kingbot.start_adventures(1000)
+kingbot.upgrade_units_smithy(village=0, units=[2,5])
+kingbot.dodge_attack(village=0, interval=300, save_resources=False, units=[3,4], target=[4,27], units_train=[1])
+kingbot.robber_hideout(village=0, interval=600, units={4:100,10:-1})
+kingbot.train_troops(village=0, units=[2,5])
+kingbot.robber_camp(village=0, interval=600, units={4:100,10:-1})
```

Comment:
could you also add some lines about these features in the readme maybe ? i think most people are looking there if they wanna know what functions there are :D

you could just copy paste what i wrote for some other functions and change parameters, nothing fancy ! just so people know this features exsits :)

Reply:
sure, I will prepare the texts!

Reply:
README updated with new features!
And I've also adjusted the parameter overload of the robber_hideouts and robber_camps methods so that the optional 'interval' is at the end of the calls.